src/lessons/06_introduction_to_javascript/introduction-to-javascript.slides.html

```diff
@@ -0,0 +1,1147 @@
+              This becomes important later
+            </aside>
+          </section>
+        </section>
+
+        <section>
+          <section>
+            <h3>Recap: Developer Tools</h3>
+
+            <p style="font-size: 0.7em">
+              Windows keyboard shortcut: <strong>F12</strong><br>
+              Mac keyboard shortcut: <strong>Command + Alt + i</strong>
+            </p>
+
+            <img data-src="images/developer-console.png" class="fragment r-frame" alt="Developer Console"/>
+
+            <aside class="notes">
+              <p>Demo: Expressions</p>
+              <ul>
+                <li>Empty expression</li>
+                <li>Constant literal</li>
+                <li>Mathematical operators (just usage, no explanation)</li>
+                <li>Return values</li>
+              </ul>
+
+              <p>Then go to slide about <strong>Variables</strong></p>
+            </aside>
+          </section>
+
+          <section>
+            <h2>Developer Console</h2>
+
+            <video data-src="images/developer-tools.mp4" controls loop data-autoplay></video>
+
+            <aside class="notes">
+              Go to slide about <strong>Variables</strong>
+            </aside>
+          </section>
+        </section>
+
+        <section>
+          <section data-auto-animate data-autoslide="2000">
+            <p>Language Basics:</p>
+            <h2>Variables</h2>
+          </section>
+
+          <section data-auto-animate>
+            <h2>Variables</h2>
+            <small>MDN: <a href="https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Grammar_and_types#declarations">Declarations</a></small>
+
+            <pre><code data-trim class="javascript" data-line-numbers="1-9 | 1-2 | 1-2,4-6 | 8-9">
+              // Variable declaration:
+              let question;
+
+              // Variable assignment:
+              question = "What's the answer to the ultimate question of life, "
+                       + "the universe and everything?";
+
+              // Declaration and assignment
+              let answer = 42;
+            </code></pre>
+
+            <aside class="notes">
+              <ul>
+                <li>Quick demo of variable declaration and assignment</li>
+                <li>Mention return values</li>
+              </ul>
+            </aside>
+          </section>
+
+          <section>
+            <h3>Two (or three) different ways of declaring variables</h3>
+
+            <pre><code data-trim class="javascript" data-line-numbers="1-5 | 1-2 | 4-5 | 1-5">
+              let myVariable = "I can be changed";
+              myVariable = "New value"; // Works fine
+
+              const myConstant = "I cannot be changed";
+              myConstant = "New value"; // Throws an error!
+            </code></pre>
+
+            <div class="additional-note fragment">
+              <h5>The old way of declaring variables:</h5>
```

Comment:
I would put it as before it was the only way, now we have the other 2. _The old way_ implies that var is not used anymore

Reply:
I want to make sure that it's clear that var and let behave differently, without going into the differences too much, because they might be a bit difficult to explain to beginners. Personally, I find var quite unintuitive at some times and believe that there is no reason to use var nowadays, unless you're still supporting Internet Explorer and don't use Babel 😅 
My idea here was therefore telling that "var" exists, in case they encounter it in third-party-code, but discouraging the students from using it.